cfme/tests/control/test_compliance.py

```diff
@@ -191,30 +165,26 @@ def test_check_package_presence(request, fleecing_vm, analysis_profile):
     )
     request.addfinalizer(lambda: diaper(profile.delete))
     profile.create()
-    fleecing_vm.assign_policy_profiles(profile.description)
-    request.addfinalizer(lambda: fleecing_vm.unassign_policy_profiles(profile.description))
-
-    with update(analysis_profile):
-        analysis_profile.categories = [
-            "check_services", "check_accounts", "check_software", "check_vmconfig", "check_system"]
-
-    do_scan(fleecing_vm)
-    assert fleecing_vm.check_compliance()
+    compliance_vm.assign_policy_profiles(profile.description)
+    request.addfinalizer(lambda: compliance_vm.unassign_policy_profiles(profile.description))
+    do_scan(compliance_vm)
+    assert compliance_vm.check_compliance()
 
 
-def test_check_files(request, fleecing_vm, analysis_profile):
+def test_check_files(request, compliance_vm, analysis_profile):
```

Comment:
This test is broken because of [DynamicTable](https://github.com/ManageIQ/integration_tests/blob/master/cfme/web_ui/__init__.py#L3601). I got some selenium errors.